new-lamassu-admin/src/pages/Cashout/helper.js

```diff
@@ -21,6 +21,12 @@ const DenominationsSchema = Yup.object().shape({
     .max(currencyMax)
 })
 
+const boldStyle = () => {
+  return {
+    fontWeight: 'bold'
+  }
+}
+
```

Comment:
If this is something that has to be common in multiple screen we should add a `boldLabel` option on the element fields and move this logic into the editableTable component